src/VisualStudio/Core/Def/Implementation/FindUsages/RepositoryColumnDefinition.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Shell.TableManager;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.FindUsages
+{
+    /// <summary>
+    /// Custom column to display the item's repository for the Find All References window.
+    /// </summary>
+    [Export(typeof(ITableColumnDefinition))]
+    [Name(ColumnName)]
+    internal class RepositoryColumnDefinition : TableColumnDefinitionBase
+    {
+        public const string ColumnName = StandardTableKeyNames.Repository;
+
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public RepositoryColumnDefinition()
+        {
+        }
+
+        public override bool IsFilterable => true;
+        public override string Name => ColumnName;
+        public override string DisplayName => ServicesVSResources.Repository;
```

Comment:
I assume we don't need to override the string content because the repository is passed to us as a user readable string?

Reply:
not certain what this means @dibarbet .  can you clarify?